File: miraculous-backend/src/main/java/com/example/miraculousbackend/dto/EventRetrievalDto.java
package com.example.miraculousbackend.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class EventRetrievalDto {

    private long id;
    private String eventTitle;
    private String eventLocation;
    private String eventTime;
    private String eventText;
    private String eventCoverPictureUrlName;
    private long artistId;

}

File: miraculous-frontend/miraculous-ui/src/dto/event-retreival-dto/event-retrieval-dto.ts
export class EventRetrievalDto {

  id: number;
  eventTitle: string;
  eventLocation: string;
  eventTime: string;
  eventText: string;
  eventCoverPictureUrlName: string;
  artistId: number;


  constructor(id?: number, eventTitle?: string, eventLocation?: string, eventTime?: string, eventText?: string, eventCoverPictureUrlName?: string) {
    this.id = id;
    this.eventTitle = eventTitle;
    this.eventLocation = eventLocation;
    this.eventTime = eventTime;
    this.eventText = eventText;
    this.eventCoverPictureUrlName = eventCoverPictureUrlName;
  }
}
